xml/ha_config_cli.xml

```diff
@@ -217,8 +232,24 @@
      </para>
     </listitem>
     <listitem>
+      <formalpara>
+        <title>As &crmsh; Cluster Scripts:</title>
+        <para>These are a collection of metadata, references to RPM packages,
+          configuration files, and &crmsh; subommands bundled under a single,
```

Comment:
typo: sucommands -> subcommands
These are managed -> They are managed (to avoid double "these")